### ISSUE TITLE
Version 1.0

### ISSUE THREAD

**Issue by soffes** [venmo/Static#63]
I'd love to get a release with #60 but that removes functionality. Should the next version be 1.0? There's a few other breaking changes I'd like to make before we 1.0 (order of initializer params, maybe remove UUID from Row).

Anyway, 1.0 feels like such a big commitment. Are we okay making a breaking change before 1.0 (like CocoaPods :P) and just do 0.3.0 with what's in master and defer the big commitment of 1.0?

What do you think @eliperkins @ayanonagon @hyperspacemark (and anyone else)?

**Comment by eliperkins** [comment]
Happy New Year! :confetti_ball: 

I'm definitely on board with getting to 1.0 soon. I'd love to come to a solution for #23 as well for 1.0, but again, not mission critical, more just a nice to have, but definitely one that'd break our existing API. My big motivation behind this is making `Row` play nice with things like text fields and passing information into `Row` objects, such as backing models. Knowing that any change around this context will break a decent amount of existing consumers is my only hesitancy on tagging this a 1.0.

Beyond any further discussion around #23 specifically, I'd say I'm all on board for getting 1.0 out the door soon. I think we're getting to a mature enough point with this that I'd feel confident tagging a 1.0 version!

**Comment by aashishdhawan** [comment]
:+1:

**Comment by eliperkins** [comment]
Chatting with @hyperspacemark a bit offline, starting to gameplan a bit what this release might look like.

We decided that adding support for table view clearing to `TableViewController` (see #50) is probably a Good Thing™ to get into 1.0 since we've committed to keeping our implementation of `TableViewController` around long enough to support iOS 8.0. Version 2.0 will drop iOS 8, using iOS 9 as a deployment target, and thus dropping this custom `TableViewController`, relying on an implementation similar to #62.

Additionally, we'd like to see Version 2.0 include support for a more generic, protocol-y `Row`/`RowType` to address #23. As such, I've opened up a milestone for Version 2.0 and we'll work towards getting future changes into that release.

@hyperspacemark mentioned he'd submit a PR to address #50. After this, we'll tag a v1.0.0 from master!
